### ISSUE TITLE
Support for dark mode

### ISSUE THREAD

**Issue by alanmeier** [yangyxd/flutter_picker#111]
Hi there,

Loved the versatility of the picker.

When testing I toggled dark mode the widget stayed the same color. Does it not support dark mode ?

Thanks

**Comment by yangyxd** [comment]
Yes, there is no contention to deal with the dark mode. You can consider adding it later. thank you!

**Comment by rh-id** [comment]
hello @yangyxd does this plugin didn't support dark mode? currently when I toggle dark mode the text stays black color and the background stays white color. i could only change the picker backgroundColor but cannot change the text color

**Comment by rh-id** [comment]
update: it works but need to set both `backgroundColor` and `textStyle`
```
Picker(
backgroundColor: darkModeOn ? Colors.black : Colors.white,
textStyle: Theme.of(context).textTheme.headline6,
)
```
